crates/libcontainer/src/process/container_main_process.rs

```diff
@@ -23,12 +26,18 @@ pub fn container_main_process(container_args: &ContainerArgs) -> Result<Pid> {
     let init_chan = &mut channel::init_channel()?;
 
     let intermediate_pid = fork::container_fork(|| {
-        container_intermediate_process::container_intermediate_process(
+        let t = container_intermediate_process::container_intermediate_process(
```

Comment:
Nit: can we have a better name than just `t`? Also maybe consider inline this with the match statement below.

Reply:
:+1: inlined